libraries/standard/http/utest/http_utest.c

```diff
@@ -88,9 +101,96 @@ static _headers_t expectedHeaders = { 0 };
 static int testRangeStart = 0;
 static int testRangeEnd = 0;
 static uint8_t httpBuffer[ HTTP_TEST_BUFFER_LEN ] = { 0 };
+static const uint8_t * pValueLoc = NULL;
+static size_t valueLen = 0u;
+static HTTPResponse_t testResponse = { 0 };
+static const size_t headerFieldInRespLoc = 44;
+static const size_t headerFieldInRespLen = sizeof( "test-header1" ) - 1u;
+static const size_t headerValInRespLoc = 58;
+static const size_t headerValInRespLen = sizeof( "test-value1" ) - 1u;
+static http_parser * pCapturedParser = NULL;
+static http_parser_settings * pCapturedSettings = NULL;
+static const char * pExpectedBuffer = NULL;
+static size_t expectedBufferSize = 0u;
+static bool invokeHeaderFieldCallback = false;
```

Comment:
I was thinking that perhaps the tests should also follow C89 as well and not use bool, so we can compile with the same settings and catch contradicting things in the library code while running the tests.

Reply:
Sure, removed use of `bool` from the file